src/api/middleware/validators/company.js

```diff
@@ -62,11 +62,36 @@ const unblock = useExpressValidators([
         .custom(companyExists).withMessage(ValidationReasons.COMPANY_NOT_FOUND),
 ]);
 
+const validCompanyId = useExpressValidators([
```

Comment:
this should be named after the route which uses it e.g. `getCompanyOffers`. Finally, it should be merged with the validation below (actually, you can delete this one, and only rename the "isExistingCompany", since the validation is repeated